client/data_definition.md

```diff
@@ -0,0 +1,65 @@
+# Data Definition in a DefraDB instance
+
+Data held in a DefraDB instance is organized into [collections](#collections) of documents.  [Collections](#collections) are [local](#local-definitions) groupings of documents that share the same [globally](#global-definitions) defined shape declared by a [schema](#schemas).
+
+## Local definitions
+
+Local definitions are specific to the node you are directly working with, they are not shared with, or assumed to be the same on other nodes in the network.
+
+Splitting local elements out from the global ones allows some local customization to the way data is organized within any given node.  It also minimizes the amount of 'stuff' that must be kept consistent across the decentralized network in order to have a well behaving database.
+
+Local data definitions are always defined on the [collection](#collections).
+
+Examples include indexes, field IDs, and [lens transforms](https://docs.source.network/defradb/guides/schema-migration).
+
+## Global definitions
+
+Global definitions are consistent across all nodes in the decentralized network of nodes in the database. This is enforced by the use of things like CIDs for schema versions.  If a global definition was to differ across nodes, the different variations will be treated as a completely different definitions.
```

Comment:
suggestion: Global definitions are consistent across all nodes in the decentralized network ~of nodes in the database~

Reply:
Will do :) I wasn't terribly confident writing that last bit out, so I'm very glad you spoke up :)

- [ ] Drop extra words